tests/opentelemetry/test_context.py

```diff
@@ -74,6 +74,8 @@ def target(parent_context):
 
 
 def _subprocess_task(parent_span_context, errors):
+    import ddtrace.auto  # noqa
```

Comment:
We also need to execute `ddtrace.auto` in the main process (ex: we need to run `test_otel_trace_across_fork` in a subprocess with ddtrace-run BEFORE creating a child process, this will ensure atexit and forksafe hooks are set.